### PR TITLE
perf(aad-manifest): update aad manifest migration middleware

### PR DESCRIPTION
**Pull request by SLdragon**
1. Do not show modal dialog when open old project
2. Generate aad-manifest-change-logs.md file after migration success
3. Add learn more button

### CODE REVIEW

**Review by KennethBWSong — APPROVED** [review]

File: packages/fx-core/resource/aad-manifest-change-logs.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Teams Toolkit continues to improve your Teams application development experience. We are upgrading the Teams app project structure so that you can:
+
+1. Use AAD manifest template to customize your AAD app.
+1. Allow to add single sign-on (SSO) feature for Bot (hosting on Azure App Service), Messaging Extension, Static Launch Page, and Embed existing web App
+
+## Know about the changes we made
+After the project upgrade, there are following changes we made:
+1. Update `.fx\configs\projectSettings.json` capabilities to include TabSSO and/or BotSSO based on original project capabilities.
+1. AAD manifest template file will be added to `templates\appPackage\aad.template.json`
+1. The required resource access information in `permissions.json` file will be merged into `aad.template.json` file
```
Comment (MuyangAmigo) [inline packages/fx-core/resource/aad-manifest-change-logs.md:17]: Do we delete `permission.json` file?

Comment (SLdragon) [reply packages/fx-core/resource/aad-manifest-change-logs.md:17]: We do not delete permissions.json file as discussed to keep combability with extension after 4.18 release

Comment (MuyangAmigo) [reply packages/fx-core/resource/aad-manifest-change-logs.md:17]: Can say something like this: The required resource access information in `permissions.json` file will be merged into `aad.template.json` file and `permission.json` file will be deprecated, please customize resource access in `aad.template.json` file.

File: packages/fx-core/resource/aad-manifest-change-logs.md [hunk 2]
```diff
@@ -14,8 +14,7 @@ Teams Toolkit continues to improve your Teams application development experience
 After the project upgrade, there are following changes we made:
 1. Update `.fx\configs\projectSettings.json` capabilities to include TabSSO and/or BotSSO based on original project capabilities.
 1. AAD manifest template file will be added to `templates\appPackage\aad.template.json`
-1. The required resource access information in `permissions.json` file will be merged into `aad.template.json` file.
-1. `permissions.json` file will not be used any more, you can update permissions using `requiredResourceAccess` property in `aad.template.json` file. You can safely delete `permissions.json` file if you and your co-workers do not use Teams Toolkit extension version <= 3.7.0.
+1. The required resource access information in `permissions.json` file will be merged into `aad.template.json` file and `permission.json file` will be deprecated, please customize `requiredResourceAccess` property in `aad.template.json` file.
```
Comment (MuyangAmigo) [inline packages/fx-core/resource/aad-manifest-change-logs.md:17]: `permission.json file` to `permission.json` file?

**Review by MuyangAmigo — APPROVED** [review]
LGTM

File: packages/fx-core/resource/aad-manifest-change-logs.md (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/resource/aad-manifest-change-logs.md]: Thanks, update

File: packages/fx-core/resource/aad-manifest-change-logs.md (hunk 2 above)
Comment (SLdragon) [reply packages/fx-core/resource/aad-manifest-change-logs.md]: fixed

**Review by nliu-ms — APPROVED** [review]

**Pull request merged**